### PR TITLE
refactor help command and aliases

### PR DESCRIPTION
**Pull request by therealplato**
Old behavior:
- `/bw help` documentation was missing `deploy` *:facepalm:*
- `/bw destroy` did nothing
- No way to list vars

New behavior:
- `/bw destroy...` now works (aliasing `/bw delete`)
- `/bw setvars...` now works (aliasing `/bw setvar`)
- `/bw info repo pr-22`
  - looks up info for playbook `repo` instance `pr-22`
  - replies with a public message to slack containing age and vars
  - all possible vars for this playbook are listed
  - vars are sorted alphabetically
- `/bw help` updated with current commands
- `/bw help` formatting improved

### CODE REVIEW

**Comment by cored** [comment]
@therealplato Looks good. What do you think if we just support one way of doing an operation with just one command?

/cc @bobbytables @poopoothegorilla

**Comment by poopoothegorilla** [comment]
@cored what do you mean `we just support one way of doing an operation with just one command?` ... do you mean no aliases?

**Comment by cored** [comment]
@poopoothegorilla yeap; aliases. Why not just support one command for each operation?

**Comment by therealplato** [comment]
> Why not just support one command for each operation?

i forget them all the time and keep trying to use the wrong ones, i've seen others do the same

**Comment by cored** [comment]
@therealplato Wouldn't be better to document the commands instead and point people to the documentation?

**Comment by therealplato** [comment]
imo no, i dont like forcing the user to behave in a particular way, especially when the cost of supporting alternate behaviors is low

**Comment by cored** [comment]
@therealplato cool, then.

**Comment by poopoothegorilla** [comment]
@cored I can see your perspective here but, I think that aliases are really nice to have. It will make for a nicer user experience. 🌈 🐶

**Comment by poopoothegorilla** [comment]
+1

**Comment by cored** [comment]
:+1:

File: services/slack.go [hunk 1]
```diff
@@ -116,9 +116,12 @@ func (c *setvarCommand) playbookContainsVar(playbookID, name string) bool {
...
 
 // CommandHints slack commands help hints
-const commandHints = `/broadway help: This message
-/broadway deploy myPlaybookID myInstanceID: Deploy a new instance
-/broadway setvar myPlaybookID myInstanceID var1=val1 githash=8ad33dad env=prod`
+const commandHints = `
+*/bw deploy myPlaybookID myInstanceID*: Deploy an instance
+*/bw info myPlaybookID myInstanceID*: Display the age and playbook variables of an instance
+*/bw &lt;delete|destroy&gt; myPlaybookID myInstanceID*: Stop and remove an instance
+*/bw &lt;setvar|setvars&gt; myPlaybookID myInstanceID var1=val1 ...* : Set one or more playbook variables for an instance
```
Comment (bobbytables) [inline services/slack.go]: I have a hunch we can make this hint a dynamic,.

File: services/slack.go [hunk 2]
```diff
@@ -116,9 +116,12 @@ func (c *setvarCommand) playbookContainsVar(playbookID, name string) bool {
 }
 
 // CommandHints slack commands help hints
-const commandHints = `/broadway help: This message
-/broadway deploy myPlaybookID myInstanceID: Deploy a new instance
-/broadway setvar myPlaybookID myInstanceID var1=val1 githash=8ad33dad env=prod`
+const commandHints = `
+*/bw deploy myPlaybookID myInstanceID*: Deploy an instance
```
Comment (bobbytables) [inline services/slack.go]: Can we add a code fence to this too actually? That way we'll get monospace font in Slack.

Comment (johnnymo87) [inline services/slack.go]: What is a code fence?

Comment (bobbytables) [inline services/slack.go]: > What is a code fence?

@johnnymo87 - It's the triple backticks that surround things in markdown to give them a "code block" look.

File: services/slack.go (hunk 1 above)
Comment (therealplato) [inline services/slack.go]: absolutely, I considered it out of scope for this PR though

File: services/slack.go (hunk 2 above)
Comment (therealplato) [inline services/slack.go]: can we try this as is first? I suspect */bold command*: explanation will be more readable than `/bold command: explanation`

**Pull request merged**